SCClassLibrary/Common/Core/Boolean.sc

```diff
@@ -18,9 +18,13 @@ Boolean {
 	keywordWarnings {
 		// turn on/off warnings if a keyword argument is not found
 		_KeywordError
+		^this.primitiveFailed
+	}
+	trace {
+		// this is only available in a special debugging version of the app
```

Comment:
I'm sorry, my response was not meant to be personal. I just like the conclusion the author makes:

> When it comes to coding standards, I take what I get and I don’t get upset.
> 
> If you happen to find yourself in a position to pick your own style guide, I recommend that you ask yourself these two simple questions:
>
> +  Is there tooling that will automatically apply the style rules to my code with little to no intervention from me?
> +  Are the tools and underlying styles actively maintained and/or used by reputable organizations?
>
> If you can answer “yes” to both of those questions, then you’re good to go. Simple as that.

IMHO, we should not nitpick within PRs (and I explicitely include myself here)  but rather concentrate on getting them closed within the current standards. If there turns out to be a need for different guideline/adjustments to guidelines, we should start a separate discussion about them.